README.md

```diff
@@ -48,15 +48,56 @@ pfSense < 2.4.5:
 ansible_python_interpreter: /usr/local/bin/python2.7
 ```
 
-Modules must run as root in order to make changes to the system.  By default pfSense does not have sudo capability so `become` will not work.  You can install it with:
-```
+To set up pfSense to be managed by Ansible, ensure the `pfSense-pkg-sudo` package is installed. This package is necessary because Ansible requires root privileges to make changes, and the default pfSense setup lacks sudo capabilities, meaning `become` will not function without it. You can install the package using the following Ansible code or manually install it by navigating to System > Package Manager > Available Packages.
```

Comment:
I don't want to say that you have to install the sudo package - users could still connect to pfsense as root.